src/pages/api/autocomplete.ts

```diff
@@ -0,0 +1,52 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { z, ZodError } from "zod";
+import { MasterItem, List, Prisma, PrismaClient } from "@prisma/client";
+var stringSimilarity = require("string-similarity");
+
+export function sortByRating(input: Array<{ target: string; rating: number }>) {
+  const sorted = [...input];
+
+  sorted.sort((a, b) => {
+    if (a.rating < b.rating) {
+      return 0;
+    }
+    if (a.rating > b.rating) {
+      return -1;
+    }
+    return 0;
+  });
+  return sorted;
+}
+
+const prisma = new PrismaClient();
```

Comment:
this should be instantiated only once, at a higher level of the application and shared via exporting the constant

read more here:
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/instantiate-prisma-client#the-number-of-prismaclient-instances-matters

Reply:
Created an instance of the prisma client on the top level index.ts